ion/binarywriter_test.go

```diff
@@ -186,7 +186,7 @@ func TestWriteBinaryTimestamp(t *testing.T) {
 	nowish, _ := NewTimestampFromStr("2019-08-04T18:15:43.863494+10:00", TimestampPrecisionNanosecond, TimezoneLocal)
 
 	testBinaryWriter(t, eval, func(w Writer) {
-		assert.NoError(t, w.WriteTimestamp(NewTimestamp(time.Time{}, TimestampPrecisionNanosecond, TimezoneUTC)))
+		assert.NoError(t, w.WriteTimestamp(NewTimestamp(time.Time{}, TimestampPrecisionSecond, TimezoneUTC)))
```

Comment:
Why this change?

Reply:
Because the [binary data](https://github.com/amzn/ion-go/pull/177/files/b3422b0e231c904ff2846e832ba54a374bbd42ea#diff-a9e03e3f1ff004ab18b39215530677a79ee3745174b52160e5d525d62240b7e1R173) doesn't have nanoseconds. It was failing because calling [NewTimestamp](https://github.com/amzn/ion-go/pull/177/files/b3422b0e231c904ff2846e832ba54a374bbd42ea#diff-4bca5f01731890e285690a1055a0f4e75e03539170cfd8f3fe6c57d6e2918b78R150) with TimestampPrecisionNanosecond will set fractional precision: 9.